src/components/WinrateGraph.js

```diff
@@ -3,20 +3,32 @@ const {h, Component} = require('preact')
 const helper = require('../modules/helper')
 const setting = remote.require('./setting')
 
+let winrateGraphMinHeight = setting.get('view.winrategraph_minheight')
 let winrateGraphHeight = setting.get('view.winrategraph_height')
 
 class WinrateGraph extends Component {
     constructor() {
         super()
 
+        this.state = {
+            sidebarWinrateGraphSplit: setting.get('view.winrategraph_height'),
+            sidebarWinrateGraphSplitTransition: true
```

Comment:
Since we're not animating the winrate graph, I don't think keeping track of `sidebarWinrateGraphSplitTransition` is necessary?

These are very verbose state names. Can we simplify them a bit? E.g. just use `height` for `sidebarWinrateGraphSplit`.